spec/datetimeformat.html

```diff
@@ -38,20 +37,28 @@ <h1>ChainDateTimeFormat ( _dateTimeFormat_, _newTarget_, _this_ )</h1>
       </emu-normative-optional>
     </emu-clause>
 
-    <emu-clause id="sec-initializedatetimeformat" aoid="InitializeDateTimeFormat">
-      <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
+    <emu-clause id="sec-createdatetimeformat" type="abstract operation" oldids="sec-initializedatetimeformat,sec-todatetimeoptions">
+      <h1>
+        CreateDateTimeFormat (
+          _newTarget_: a constructor,
+          _locales_: an ECMAScript language value,
+          _options_: an ECMAScript language value,
+          _required_: a String,
+          _defaults_: a String
+        ): either a normal completion containing a DateTimeFormat object or a throw completion
```

Comment:
```suggestion
        ): a DateTimeFormat object
```

Reply:
An AO must always, or never, return a completion; if it returns a completion, the return type must indicate it.